pkgs/development/python-modules/mmcv/default.nix

```diff
@@ -0,0 +1,127 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytestCheckHook
+, pythonOlder
+, torch
+, torchvision
+, opencv4
+, yapf
+, packaging
+, pillow
+, addict
+, ninja
+, which
+, onnx
+, onnxruntime
+, scipy
+, pyturbojpeg
+, tifffile
+, lmdb
+, symlinkJoin
+}:
+
+let
+  inherit (torch) cudaCapabilities cudaPackages cudaSupport;
+  inherit (cudaPackages) backendStdenv cudaVersion;
+
+  # for cuda support, we are waiting on opencv: misc CUDA-related updates and fixes; add enableLto #221370
```

Comment:
This comment is now out of date.

Reply:
it was merged in Nixos:Staging not master. so its still relevant

Reply:
It's currently in staging-next https://nixpk.gs/pr-tracker.html?pr=221370

I would suggest to just leave this open until that is merged which will be very soon.

Reply:
#221370 has been merged into master. @SuperSandro2000

Reply:
So you can resolve the comment now?

Reply:
removed.